components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -2123,6 +2145,23 @@ private AuthenticationGraphConfig getGraphConfiguration(int applicationId, Conne
         return authenticationGraphConfig;
     }
 
+    private AuthenticationScriptConfig getScriptConfiguration(int applicationId, Connection connection)
+            throws SQLException {
+
+        PreparedStatement localAndOutboundConfigScriptPrepStmt = null;
+        ResultSet localAndOutboundConfigScriptResultSet = null;
+        String LOAD_AUTH_SCRIPT_CONFIG = "SELECT CONTENT FROM SP_AUTH_SCRIPT WHERE APP_ID = ?";
+        localAndOutboundConfigScriptPrepStmt = connection.prepareStatement(LOAD_AUTH_SCRIPT_CONFIG);
+        localAndOutboundConfigScriptPrepStmt.setInt(1, applicationId);
+        localAndOutboundConfigScriptResultSet = localAndOutboundConfigScriptPrepStmt.executeQuery();
+        if (localAndOutboundConfigScriptResultSet.next()) {
+            AuthenticationScriptConfig authenticationScriptConfig = new AuthenticationScriptConfig();
+            authenticationScriptConfig.setContent(localAndOutboundConfigScriptResultSet.getString(1));
```

Comment:
try(Reader reader = localAndOutboundConfigScriptResultSet.getCharacterStream(1)){
                String targetString = IOUtils.toString(reader);
                authenticationScriptConfig.setContent(targetString);
            } catch (IOException e) {
                log.error("Could not read the Script for application : "+applicationId, e);
                //TODO: Handle and throw proper exception
                return null;
            }